redhat/python-exabgp.spec.git

```diff
@@ -1,20 +1,29 @@
 %{!?__python2:        %global __python2 /usr/bin/python2}
 %{!?python2_sitelib:  %global python2_sitelib %(%{__python2} -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())")}
+%define version %(echo "$(python setup.py current).git$(git rev-parse --short HEAD)")
```

Comment:
Executing in a shell:

![captura de tela de 2017-05-12 15-45-08](https://cloud.githubusercontent.com/assets/201189/26012169/0d74e31e-372a-11e7-8005-668eac97ccfe.png)

The puporse of this line is populate a version numer with you corrected syntax + git<COMMIT> recommendation from Fedora - as in https://fedoraproject.org/wiki/Packaging:Versioning#Snapshots